### DISCUSSION TITLE
PHP8.4

### DISCUSSION THREAD

**Discussion by s-ohnishi** [phalcon/cphalcon#16665] [General]
PHP 8.4 will be released soon, but will Phalcon be compatible with it?

**Comment by Jeckerson** [comment]
Sure, after its release, Zephir will be adapted to support it and only then Phalcon will be recompiled to support PHP8.4. We will need to wait until release, as there are another extensions that are updated only after official release.

**Reply by s-ohnishi** [reply]
I feel relieved. Thank you.
